src/dmd/dcast.d

```diff
@@ -2966,6 +2966,20 @@ Lagain:
     t1 = t1b;
     t2 = t2b;
 
+    bool tryToMergeAliasThis(Scope *sc, Type *pt, TOK op, Expression *pe1, Expression *pe2)
```

Comment:
I think this function should be `static` also.

Reply:
Nope, because it modifies local variables of outer function.